### ISSUE TITLE
Customer Account block should render user avatar when available

### ISSUE THREAD

**Issue by Aljullu** [woothemes/woocommerce#42449]
WordPress has a [`get_avatar()`](https://developer.wordpress.org/reference/functions/get_avatar/) function which allows to load the user avatar. That avatar comes either from Gravatar or from storing it in the site with plugins like [Simple Local Avatars](https://wordpress.org/plugins/simple-local-avatars/).

I think it would be nice if the Customer Account block rendered the user avatar when available. Maybe making this as an option?

## To reproduce

1. Make sure your user email has a profile picture in Gravatar (alternatively, install Simple Local Avatars and add an avatar to your account).
2. Add the Customer Account block to your store header.
3. Log in and notice the avatar is not visible anywhere.

<img src="https://user-images.githubusercontent.com/3616980/223976877-367df21c-cdc2-46eb-8982-1c5e5c240f2d.png" alt="" width="347" />

## Expected behavior

If the user has an avatar, the Customer Account block should use it.

**Comment by kkoteen** [comment]
Routing this to design for the next step: @yjailin would love your feedback on this. 

@Aljullu, any extra context or thoughts since you first filed this issue?

**Comment by Aljullu** [comment]
> @Aljullu, any extra context or thoughts since you first filed this issue?

Not really, I think the issue is still valid. As long as product and design agree with it, it shouldn't be too difficult to implement from the engineering side.

**Comment by yjailin** [comment]
I don't see any issue doing that. As long as the avatar size is controlled. Did some quick test in Figma! @kkoteen @Aljullu 
fpl1YUEIW7suBFO09qymA2-fi-7896_109179

<img width="1444" height="831" alt="Image" src="https://github.com/user-attachments/assets/86c9bccd-5d00-45b0-9717-2f014eb44e01" />

**Comment by Aljullu** [comment]
Thanks, @yjailin! As I'm working on this in #67032, one question: do you have a preference on whether this should be a block attribute that users can enable/disable or it should be applied to the block in all cases? Users can still disable avatars from Settings > Discussion > Show Avatars, so in theory a block attribute wouldn't be needed, but if we enable this feature in all block instances, it will affect existing stores using this block. Are we fine with that?

**Comment by yjailin** [comment]
@Aljullu I'd go always-on, no block toggle. Avatars only show when the user is logged in, has an avatar, and the store has avatars enabled globally, so it already respects the Settings > Discussion switch. A block attribute would just duplicate that.

**Comment by Aljullu** [comment]
Great, it makes sense. Thanks, @yjailin!
